db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,33 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :deployment_authentications do |t|
```

Comment:
@deads2k 
* It's not intended for mulitple identity providers, we can make it work though, is that a priority? (plus current UI doesn't support multi so we will need to change that as well)
* BasicAuthPasswordIdentityProvider wasn't listed as one of the identety providers we will support, is that a priority?
* the validation will be made in the UI.